en/docs/learn/api-security/oauth2/grant-types/jwt-grant.md

```diff
@@ -1,69 +1,137 @@
 # JWT Grant
 
-The JSON Web Token(JWT) bearer grant is simply a JSON string containing claim values that will be evaluated and validated by the JWT Grant Handlers at the Authorization Server end, before issuing an access token.
+The JSON Web Token (JWT) Bearer Grant is simply a JSON string containing claim values that will be evaluated and validated by the JWT Grant Handlers at the Authorization Server end, before issuing an access token.
 
-WSO2 API Manager, as an OAuth 2.0 Authorization Server with its key manager features, can accept JWT Assertions from OAuth 2.0 clients as means of resource owner authentication and authorization. Additionally, it can exchange the JWT access token with OAuth 2.0 access tokens in order to access protected resources on behalf of the resource owner.
+WSO2 API Manager (WSO2 API-M), as an OAuth 2.0 Authorization Server with its key manager features, can accept JWT Assertions from OAuth 2.0 clients as means of resource owner authentication and authorization. Additionally, it can exchange the JWT access token with OAuth 2.0 access tokens in order to access protected resources on behalf of the resource owner.
 
 -   [Configuring the JWT grant](#configuring-the-jwt-grant)
 -   [Using the JWT grant](#using-the-jwt-grant)
 -   [JWT Bearer Grant](#jwt-bearer-grant)
 
-### Configuring the JWT grant
+## Configuring the JWT grant
 
-1.  Sign in to the WSO2 API Manager. Enter your username and password to log on to the Management Console (<https://localhost:9443/carbon>).
+### Step 1 - Obtain a JWT from an external Identity Provider 
 
-2.  Navigate to the **Identity Providers** section under the **Main** tab of the management console and click **Add.**
-3.  Provide the following values to configure the IDP:
-    -   **Identity Provider Name:** Enter an issuer name as the identity provider name. This is used to generate the JWT assertion.
-    -   **Identity Provider Public Certificate :** The certificate used to sign the JWT assertion.
+You can use any identity provider to obtain a JWT. As an example, this step will discuss how to obtain a JWT from WSO2 Identity Server (WSO2 IS).
 
-    !!! info
-        Identity provider Public Certificate
+1. Download and install the [latest WSO2 Identity Server](https://wso2.com/identity-and-access-management/#).
+     
+     If you downloaded the archive, extract it. `<IS_HOME>` refers to the root folder of the extracted WSO2 IS.
 
-        The **Identity Provider Public Certificate** is the public certificate belonging to the identity provider. Uploading this is necessary to authenticate the response from the identity provider.
+     Start WSO2 Identity Server with a port offset. (`portOffset` is required only if you are running both API-M and ID in the same) JVM.
 
```

Comment:
Parenthesis has to be after JVM?

Reply:
Fixed via https://github.com/wso2/docs-apim/pull/3170/commits/d99c31c64db63b3df9a1aad0bc9edde2c10379f0